models/evc.py

```diff
@@ -647,8 +647,8 @@ def get_path_status(path):
 
     def remove(self):
         """Remove EVC path and disable it."""
-        self.remove_current_flows()
-        self.remove_failover_flows()
+        self.remove_current_flows(sync=False)
```

Comment:
Sweet. Appreciated you proactively also fixing this adjacent issue #416

Reply:
Added issue to main comment.